src/validation/fields/date.validation.ts

```diff
@@ -31,7 +32,28 @@ export const start_date_validations = [
   body("start_date-year")
     .custom((value, { req }) => checkDateFieldYear(ErrorMessages.YEAR, ErrorMessages.YEAR_LENGTH, req.body["start_date-day"], req.body["start_date-month"], req.body["start_date-year"])),
   body("start_date")
-    .custom((value, { req }) => checkStartDate(req.body["start_date-day"], req.body["start_date-month"], req.body["start_date-year"])),
+    .custom((value, { req }) => checkDate(req.body["start_date-day"], req.body["start_date-month"], req.body["start_date-year"])),
+];
+
+export const ceased_date_validations = [
+  body("ceased_date-day")
+    .if(body('is_still_bo').equals('0'))
+    .custom((value, { req }) => checkDateFieldDay(req.body["ceased_date-day"], req.body["ceased_date-month"], req.body["ceased_date-year"])),
```

Comment:
we can chain two or more custom() together.

Reply:
maybe for line 48 on

Reply:
Have changed the last two customs as they operate on "ceased_date" field